newsletter-december-2022.rst

```diff
@@ -0,0 +1,132 @@
+.. post:: Dec 5, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - December 2022
+========================================
+
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-december-2022>`.
+
+
+News and updates
+----------------
+
+The latest updates from our team:
+
+- 🐘️ We started *tooting* in the Fediverse (Mastodon). Follow us at `@readthedocs@fosstodon.org <https://fosstodon.org/@readthedocs>`_
+- 🎬️ Visit the upcoming landing page `about.readthedocs.com <https://about.readthedocs.com/>`_ and `let us know <mailto:hello@readthedocs.org>`_ know if you have comments or ideas for it.
+- ⚙️ The *entire* build process can now be overridden (if you need it). :doc:`See the Announcement </build-customization>`.
+
+  - 💡️ ...this is useful if you want to publish outputs generated from a framework that isn't supported by default. We wrote some examples for :ref:`Pelican <readthedocs:build-customization:Pelican>` and :ref:`Docsify <readthedocs:build-customization:Docsify>`.
+  - 💡️ ...you can also *extend* the build process for instance you can :ref:`skip a build <readthedocs:build-customization:Cancel build based on a condition>`.
+
+- ⬇️ Sphinx 6 is coming soon, more details :ref:`further down <sphinx6_is_coming>`.
+- 🛳️ :doc:`Server-Side Search API v3 <readthedocs:server-side-search/index>` has been released.
```

Comment:
This is too probably too hidden, but we should promote this work more in general.

Reply:
It'll seem less hidden once the Sphinx 6 part is removed... maybe moving it up can help, too.

Reply:
Moved a bit, but it would require some more work to make it more featured.

Thoughts?

Reply:
I think it's fine, it's a longer term thing to highlight it more.

Reply:
Ah yes, agreed, more documentation articles documenting how to benefit from it could maybe be a way forwards.